README.md

```diff
@@ -6,76 +6,76 @@
 [![Transifex](https://img.shields.io/badge/transifex-en|nl|sv|no|fi-blueviolet)](https://www.transifex.com/retrospect/public/)
 
-This repository holds the main code for Retrospect. For more information on bug reporting, please visit https://github.com/retrospect-addon/plugin.video.retrospect or https://www.rieter.net/content/.
+This repository holds the main code for Retrospect. For more info on reporting bugs, please visit https://github.com/retrospect-addon/plugin.video.retrospect or https://www.rieter.net/content/.
 
 # Introduction #
 Retrospect (Previously XBMC Online TV) is a Kodi video add-on which allows you to watch streams of 
-a number of free and publicly available online TV stream sites.
+a number of gratis publicly available online TV stream sites.
 
-It originally started the streams from [www.uitzendinggemist.nl](https://npostart.nl) which is a 
-Dutch site. The add-on got its initial name from this site: ‘_Uitzending gemist_’ means 
+It all started with the streams from the Dutch site [www.uitzendinggemist.nl](https://npostart.nl).
+The add-on got its initial name from this site: ‘_Uitzending gemist_’ meaning 
 ‘_missed broadcast_’ in Dutch.
 
-Discussion about the add-on can be done in [this](https://forum.kodi.tv/showthread.php?tid=25522) 
-thread at the Kodi forums or in [this](https://gathering.tweakers.net/forum/list_messages/1643928/last) 
-thread at Tweakers.net (Dutch only). Issues regarding this script can be submitted at our issue 
-tracker at GitHub.
+Discuss the add-on can be done at [the Kodi forums thread](https://forum.kodi.tv/showthread.php?tid=25522),
+or at [the Tweakers.net thread](https://gathering.tweakers.net/forum/list_messages/1643928/last) (Dutch only).
+Issues regarding this script can be submitted at our issue tracker at GitHub.
 
 # Installing Retrospect #
-There are two ways to install Retrospect, depending on what version of Kodi you are using.
+There are two ways of installing Retrospect, (depending on the Kodi version you are using).
 
 ### Kodi Leia and later
-Starting from Kodi Leia (v18), you can easily install Retrospect from the official Kodi add-on repository. Simply use the search function in the add-ons section to find `Retrospect` and install it. More detailed information can be found in the [Retrospect Wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
+Starting from Kodi Leia (v18), you can install Retrospect from the official Kodi add-on repository. Use the search function in the add-ons section to find `Retrospect` and install it. Detailed info can be found in the [Retrospect wiki](https://github.com/retrospect-addon/plugin.video.retrospect/wiki/Installation).
 
-**Note:** Make sure the _Auto-Update_ option for Retrospect is enabled to automagically receive new updates.
+**Note:** Make sure the _Auto-Update_ option for Retrospect is turned on to automagically receive new updates.
```

Comment:
It really is `enabled`, that is how it is called for almost all Kodi (and add-on) settings.

Reply:
Unless it can be turned on, and disabled at the same time, semantically you are adding complexity. Also makes it sound less welcoming. I put in "Turn on/off" for Kodi stuff when I can.